tests/test_ndmg_dwi_pipeline.py

```diff
@@ -6,10 +6,91 @@
 import ndmg
 from ndmg.utils.cloud_utils import s3_get_data
 from pathlib import Path
+from ndmg.utils.gen_utils import create_datadescript, DirectorySweeper
+import tempfile
```

Comment:
I think you don't use this import? maybe I'm just not seeing it

Reply:
You're right, this was a leftover from a different method I used to make the files, thought it was necessary for `tmp_path` but that isn't true